release/scan_docker_images.sh

```diff
@@ -119,6 +119,8 @@ function scan_docker_images {
 
 		lc_log INFO "${scan_output}"
 
+		rm --force ./twistcli
+
```

Comment:
@lucasmiranda0, if we define more than one Docker image to be scanned in the variable `LIFERAY_IMAGE_NAMES`, shouldn't we remove `twistcli` later?

Otherwise, we would scan just the first Docker image only.